packages/grafana-ui/src/components/PanelChrome/PanelLinks.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { LinkModel } from '@grafana/data';
+
+import { Dropdown } from '../Dropdown/Dropdown';
+import { Menu } from '../Menu/Menu';
+import { ToolbarButton } from '../ToolbarButton';
+
+interface Props {
+  links: (() => LinkModel[]) | undefined;
+}
+
+export function PanelLinks({ links }: Props) {
```

Comment:
Panel links as drop-down menu is a change from the tooltips we use today, is there a reason to change them from tooltips (tooltips don't require clicks)

Reply:
Yes, I want to make the links accessible via the keyboard, so I decided to use the `Dropdown` component. Unfortunately, this component only supports `click` to display the menu, there is no `hover` option :(.

Reply:
I think each link should be it's own icon and be a href

Reply:
🤔 do you mean that if we configure 3 panel links, the panel header should have 3 of these icons?

![image](https://user-images.githubusercontent.com/239999/211762508-94a08ef0-f119-43b2-bafb-13449669e06a.png)

If yes, I believe this can make very crowded the panel header, as we don't have a limit on panel links, also it is not aligned with the current design.

Reply:
@axelavargas yes, that is one of the benefits of this new design is to give more space and make links more accessible so you can access them without hovering over a tooltip. 

But true, you can have multiple. And if more than one maybe collapsing them into a menu is better

Reply:
I have followed the approach you recommended @dprokop , it looks way better 🙌🏾, [here relevant commit](https://github.com/grafana/grafana/pull/60514/commits/e445a727e25dc574ea2bb955050234d4894a890c)